src/native/chunk/splitter.cpp

```diff
@@ -44,8 +44,9 @@ Splitter::Splitter(
     if (_calc_md5) {
         extern bool fips_mode;
         if (fips_mode) {
-            posix_memalign((void**)&_md5_mb_mgr, 16, sizeof(MD5_HASH_CTX_MGR));
-            posix_memalign((void**)&_md5_mb_ctx, 16, sizeof(MD5_HASH_CTX));
+            [[maybe_unused]] int result;
+            result = posix_memalign((void**)&_md5_mb_mgr, 16, sizeof(MD5_HASH_CTX_MGR));
+            result = posix_memalign((void**)&_md5_mb_ctx, 16, sizeof(MD5_HASH_CTX));
```

Comment:
why do we set it into a variable (that we are not using?)

Reply:
We need to handle the return value of `posix_memalign()` if don't do that than it gives a warning which is being treated as an error on CI job.